open_schools_platform/common/firebase/settings.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+import firebase_admin
+
+from firebase_admin import credentials
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+FIREBASE_API_KEY = os.path.join(BASE_DIR, "firebase_key.json")
+
+cred = credentials.Certificate(FIREBASE_API_KEY)
+app = firebase_admin.initialize_app(cred)
```

Comment:
I think we can locate this logic to utils dircetory 
instead of common directory